x-pack/plugins/observability/public/plugin.ts

```diff
@@ -55,6 +55,7 @@ export interface ObservabilityPublicPluginsSetup {
 }
 
 export interface ObservabilityPublicPluginsStart {
+  usageCollection: any;
```

Comment:
Looking at the contract in `usageCollection`, you need the `setup` contract to reach out to `ApplicationUsageTrackingProvider`.

Reply:
Yep I was guessing something like that. Will give it a try. Thanks!